apps/app/src/views/routes/PatientForm.tsx

```diff
@@ -22,7 +22,7 @@ export const PatientForm = () => {
         if (e?.detail?.createPrescription) {
           navigate(`/prescriptions/new?patientId=${id}`);
         } else {
-          navigate(`/patients?reload=${id}-${ulid()}`);
+          navigate(`/patients`);
```

Comment:
Do we know why we had those query params? 
@jknapp25 @cawfeecoder

Reply:
I used to remember - I think it had to do with how routing was setup to make sure that we'd reload the list that you get kicked back to on save/updated (since one is a webcomponent and the other is a React element)

Reply:
From what I can tell it's not filtering the list in any way, it's the entire list either way

Reply:
When I update a patient I'm seeing the changes reflected (though I can't update a name... should I be able to?)

https://www.loom.com/share/b400b1d631cc4aea882311b27b2f9516